<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="2" hostname="DESKTOP-LC06BTB" name="com.training.sanity.tests.RealestateMediumTests" tests="4" failures="1" timestamp="29 Nov 2019 04:36:16 GMT" time="168.917" errors="0">
  <testcase name="adminLogin" time="76.403" classname="com.training.sanity.tests.RealestateMediumTests">
    <failure type="java.lang.AssertionError" message="expected [Dashboard ‹ Real Estate — WordPress] but found [My Profile – Real Estate]">
      <![CDATA[java.lang.AssertionError: expected [Dashboard ‹ Real Estate — WordPress] but found [My Profile – Real Estate]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertEqualsImpl(Assert.java:134)
	at org.testng.Assert.assertEquals(Assert.java:115)
	at org.testng.Assert.assertEquals(Assert.java:189)
	at org.testng.Assert.assertEquals(Assert.java:199)
	at com.training.pom.LoginPOM.verifyadminlogin(LoginPOM.java:86)
	at com.training.sanity.tests.RealestateMediumTests.adminLogin(RealestateMediumTests.java:64)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:523)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:224)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:599)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- adminLogin -->
  <testcase name="RETC_031" time="92.514" classname="com.training.sanity.tests.RealestateMediumTests"/>
  <testcase name="RETC_032" time="0.000" classname="com.training.sanity.tests.RealestateMediumTests">
    <skipped/>
  </testcase> <!-- RETC_032 -->
  <testcase name="RECT_033" time="0.000" classname="com.training.sanity.tests.RealestateMediumTests">
    <skipped/>
  </testcase> <!-- RECT_033 -->
</testsuite> <!-- com.training.sanity.tests.RealestateMediumTests -->
